### PR TITLE
FBI graphql changes  added

### PR DESCRIPTION
**Pull request by spaceo**
#### Link to issue

https://reload.atlassian.net/browse/DDFSOEG-165

#### Description

After the graphql API changed to the new FBI API we needed to change queries, types and functionalities in the autosuggest and search result part of the code

#### Checklist

- [x] My complies with [our coding guidelines](../docs/code_guidelines.md).
- [x] My code passes our static analysis suite. If not then I have added a comment explaining why this change should be exempt from the code standards and process.
- [x] My code passes our continuous integration process. If not then I have added a comment explaining why this change should be exempt from the code standards and process.

#### Note
Old demo components/apps were deleted as well since we do not need them anymore.

### CODE REVIEW

File: src/components/search-result-list/search-result-list-item/search-result-list-item-series.tsx [hunk 1]
```diff
@@ -2,17 +2,19 @@ import React from "react";
...
 interface SearchResultListItemSeriesProps {
-  series: SeriesSimpleFragment;
+  series: SeriesSimpleFragment[];
 }
 
 const SearchResultListItemSeries: React.FC<SearchResultListItemSeriesProps> = ({
-  series: { part, title }
+  series
 }) => {
+  const display = series?.[0]?.numberInSeries?.display;
```
Comment (Adamik10) [inline src/components/search-result-list/search-result-list-item/search-result-list-item-series.tsx:11]: I maybe would call this const something that is meaningful. In the data, it is called display, that's right, but I would maybe suggest calling it something like numberInSeries or stng ike that.

File: src/core/fbs/mutator/fetcher.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { getToken, TOKEN_USER_KEY } from "../../token";
+import { getToken, TOKEN_LIBRARY_KEY, TOKEN_USER_KEY } from "../../token";
```
Comment (Adamik10) [inline src/core/fbs/mutator/fetcher.ts:1]: @spaceo what about these GitHub warnings?
'TOKEN_USER_KEY' is defined but never used

**Review by Adamik10 — CHANGES_REQUESTED** [review]
Very good job 🥇 You've done well with this mildly annoying API change task. I have a couple of comments + one that i couldn't comment in connection to the file -> src/core/dbc-gateway/CreatorSimple.fragment.graphql -> this file is empty. What is it?

File: src/components/search-result-list/search-result-list-item/search-result-list-item-series.tsx (hunk 1 above)
Comment (spaceo) [reply src/components/search-result-list/search-result-list-item/search-result-list-item-series.tsx:11]: 👍

File: src/core/fbs/mutator/fetcher.ts (hunk 2 above)
Comment (spaceo) [reply src/core/fbs/mutator/fetcher.ts:1]: 👍

**Review by cableman — APPROVED** [review]

**Pull request merged**